calico_containers/calico_ctl/utils.py

```diff
@@ -48,15 +48,20 @@ def check_ip_version(ip, version, cls):
     :return: The parsed object of type "type"
     """
     assert version in ("v4", "v6")
+    parsed = None
     try:
         parsed = cls(ip)
```

Comment:
I mentioned in a comment in the tests that I should have caught this before, but I think you should change the name of the `cls` because `cls` is used a keyword in python.  Something like ip_cls or similar would be good.